source/nodejs/adaptivecards-designer/src/shared.ts

```diff
@@ -14,6 +14,6 @@ export class GlobalSettings {
         Adaptive.Versions.v1_3,
         Adaptive.Versions.v1_4,
         Adaptive.Versions.v1_5,
-        Adaptive.Versions.v1_6
+        Adaptive.Versions.v1_6,
```

Comment:
>Adaptive.Versions.v1_6, [](http://example.com/codeflow?start=8&length=23)

nit: extra change

Reply:
trailing commas are recommended in my old project. It has some benefit, and there is tslint rule for this.
I expect we didn't setup the tslint correctly in our project.
https://stackoverflow.com/questions/52218260/tslint-missing-trailing-comma-trailing-comma-on-the-last-line

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas